code/PostgreSQLQuery.php

```diff
@@ -80,7 +80,11 @@ public function nextRecord()
                 $record[$k] = $v;
                 $type = pg_field_type($this->handle, $i);
                 if (isset(self::$typeMapping[$type])) {
-                    settype($record[$k], self::$typeMapping[$type]);
+                    if ($type === 'bool' && $record[$k] === 't') {
+                        $record[$k] = 1;
+                    } else {
+                        settype($record[$k], self::$typeMapping[$type]);
```

Comment:
Does this implicitly handle `f` values?

Reply:
Yeah settype('f', 'int') returns `0`. Is it too hacky to rely on that?

Reply:
Maybe a comment? I was just curious